plugin/flags_sources/cmake_file.py

```diff
@@ -42,7 +42,8 @@ def __init__(self,
                  prefix_paths,
                  flags,
                  cmake_binary="cmake",
-                 header_to_source_mapping=None):
+                 header_to_source_mapping=None,
+                 use_compiler_builtins=False):
```

Comment:
Let's probably remove the default value. This will ensure that we are always propagating the settings correctly. I will then add a separate PR to remove other default values here. They should not be there. It just adds a new point of breakage. :)